templates/deb_importer.json.erb

```diff
@@ -1,4 +1,5 @@
 {
+    "remove_missing": true,
```

Comment:
Should this be configurable?

Reply:
Not if i can avoid it. ;)
See also:
https://github.com/Katello/katello/pull/7773